integrations/ingress-nginx/documentation.yaml

```diff
@@ -0,0 +1,90 @@
+exporter_type: included
+app_name_short: Ingress NGINX
+app_name: {{app_name_short}}
+app_site_name: Ingress NGINX
+app_site_url: https://kubernetes.github.io/ingress-nginx
+exporter_name: the Ingress NGINX exporter
+exporter_pkg_name: ingress-nginx
+exporter_repo_url: https://kubernetes.github.io/ingress-nginx/user-guide/monitoring/
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. You can run the following checks to verify
+  that {{app_name_short}} is exposing Prometheus-format metrics.
+
+  To determine if {{exporter_name}} is exposing metrics, run the following command:
+
+  <pre class="devsite-click-to-copy">
+  kubectl exec -n {{ namespace }} deploy/ingress-nginx-controller -- curl http://localhost:10254/metrics
+  </pre>
+dashboard_available: true
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: ingress-nginx
+    labels:
+      app.kubernetes.io/name: ingress-nginx
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+    - port: 10254
+      scheme: http
+      interval: 30s
+      path: /metrics
+    selector:
+      matchLabels:
+        app.kubernetes.io/name: ingress-nginx
+additional_podmonitoring_info: |
+  Ensure that the port and matchLabels match that of the {{app_name_short}} pods you wish to monitor. By default,
```

Comment:
the values of the `port` and `matchLabels` fields match those  ... you want to monitor.

(We try to avoid "wishing" in docs. If you could make things work by just wishing, life would be soooo much easier!)